### ISSUE TITLE
The top hit is Glipizide. Under evidence in publications the top hit is a 2021 article about repurposing drugs to treat COVID19. The paper seems kind of old and sort of off target for the top hit for this drug. For example KEGG entry is :  https://www.genome.jp/entry/D07118 summarizing more info I'm likely to want. MOI, etc.. I wonder if the top literature hit could be a review if one is available? --Maureen

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/Feedback#416]

**Comment by andrewsu** [comment]
**TLDR: I think this is either a UI issue, or at least we need the help of someone on the UI team to work back to the underlying issue.**  

----------

These are the "Sources" that support `Glipizide - treats - T2D`:

![image](https://github.com/NCATSTranslator/Feedback/assets/2635409/e21e28aa-b2b2-44d8-a3f5-a8089ce8d29e)

I can see three ARAs in this list -- Aragorn, ARAX, and BTE.  When I look at those ARA results at https://arax.ci.transltr.io/?r=ef653aa8-7263-48df-b0ac-5a0b9a1344e8:
* Aragorn: ARS reports a 598 error for Aragorn, so I'm not sure why/how it is being reported by the UI as a source
* ARAX: Does not appear to have Glipizide among the 10 reported results, so also not sure why it's being reported by the UI as a source
* BTE: Glipizide is result 23, but [PMC7298882](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7298882/) is not cited in the evidence, either on the direct edge nor on the edges in the supporting graphs

So I'm not sure how to work back to where this issue originated.

**Comment by dnsmith124** [comment]
Looking at the data being sent to the UI I can confirm that there's no error with the UI itself. The PMC in question comes in attached to the edge "Glisoxepide Treats Type 2 Diabetes Mellitus", which is the second part of a 2 hop path for the result Glipizide.  

Generally checking the specific edge makes it easier to debug these sorts of situations, rather than looking at the entire set of evidence for a result. If we check the specific edge this evidence is associated with in the path view (Glisoxepide Treats Type 2 Diabetes Mellitus), there's only 1 pub (the weird one related to covid) and 3 sources: 

![Screen Shot 2023-07-21 at 1 26 03 PM](https://github.com/NCATSTranslator/Feedback/assets/54226761/98c40eb1-e230-4c58-94c4-b444d3a4a53e)

If I take the original pk and put it into the dev instance of the arax ui I can see the edge and publication came from Robokop. (Path to get there: Load robokop results, select result 5 Glipizide, select treats edge with support graph, select first support graph ending in 53f5, select first Glisoxepide treats type 2 diabetes mellitus edge):

![Screen Shot 2023-07-21 at 1 46 40 PM](https://github.com/NCATSTranslator/Feedback/assets/54226761/e70c3786-28cc-41a8-8ea9-05616bc16b8e)

All of this looks legitimate to me, as the text mentions the drug in question is used for treatment of diabetes. The paper itself isn't really relevant to the edge (at least not directly), so it may want to be tossed, but there's no bug here as far as I can tell. 

I think a big part of the of the confusion here is assuming that this publication shows up as the "top hit", as though it's being sorted by some sort of relevancy factor, when in fact it's merely the most recently published publication out of all the publications for that result (which is the default sorting method in the evidence window, indicated by the tick mark in the relevant column header).

**Comment by sstemann** [comment]
@dnsmith124 thank you for investigating, looks like the root cause is from Robokop, and I can't reproduce it in CI (ie i don't see the unrelated pub on those edges)

@Genomewide @sierra-moxon is there a relevancy sort that can be done on the EPC of the pubs?

**Comment by sierra-moxon** [comment]
Masquerading as a user, when I click on "View All Evidence" I expect the pubs to be directly related to the "drug _treats_ disease" edge.  

Can we put all the pubs that are reported on the "treats" edge of the summary/first path (in this case, a one-hop "treats" inferred edge), into the first page of "View All Evidence" popup?  This way, at first glance by the user, the "treats" edge publications will appear first in the "View All Evidence" display for the entire result.

**Comment by Genomewide** [comment]
I think there are def some ways to help sort publications, like highlighting if a publication shows up in a result multiple times or being able to see if it is a review. 

Also def post september

**Comment by sstemann** [comment]
This is now the first pub you can view when you run Type 2 Diabetes Mellitus in Prod

https://ui.transltr.io/demo/results?l=Type%202%20Diabetes%20Mellitus&i=MONDO:0005148&t=0&r=0&q=53103a99-c1aa-49e7-acb0-0999e5af80df

Its from the Curated CTD source
<img width="815" alt="image" src="https://github.com/user-attachments/assets/c3638723-a1ac-4f17-b472-5876a3b567c3">

If you click the pub title you end up here:  https://pubmed.ncbi.nlm.nih.gov/7782488/
<img width="562" alt="image" src="https://github.com/user-attachments/assets/e735748e-b8a7-464d-b3c0-e929415b391d">

Via CTD > Chemical Search 
<img width="940" alt="image" src="https://github.com/user-attachments/assets/4056804a-fe0f-4f16-a435-2785b244ccfa">

Click its reference and you get.  SO it seems the ingest is fine.  The fact that it's a very highly rated result with 500 publications and THIS is the first publication you get to, seems odd.  But I'm not sure what the UI would use to rank the pubs and if the ARAs say its a good ingest, then I think we are stuck.  

<img width="958" alt="image" src="https://github.com/user-attachments/assets/e1a391c6-96ae-43fa-b8b6-63042d7be8a8">

On the backend, at least one instance of this is MolePro via ARAX

<img width="819" alt="image" src="https://github.com/user-attachments/assets/e13baca0-eb98-4804-a4ef-44fb6410431d">
